kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -25,7 +25,7 @@ domain.weblogic.oracle/domain1 patched
 [INFO] Successfully patched server 'managed-server2' with 'ALWAYS' start policy.
 ```
 
-The `stopServer.sh` script shuts down a running Managed Server either by decreasing the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or by patching the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
+The `stopServer.sh` script shuts down a running WebLogic Server in a domain. For clustered Managed servers, it either decreases the `spec.clusters[<cluster-name>].replicas` value for the Managed Server's cluster by `1` or updates the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource or both as necessary. For the administration server, it updates the value of `spec.adminServer.serverStartPolicy` attribute of the domain resource. For non-clustered Managed servers, it updates the `spec.managedServers[<server-name>].serverStartPolicy` attribute of the domain resource. The script provides an option to keep the `spec.clusters[<cluster-name>].replicas` value constant for clustered servers. See the script `usage` information by using the `-h` option.
```

Comment:
Managed servers -> Managed Servers
it either increases -> either it increases
administration server -> Administration Server
value of -> value of the
Managed servers -> Managed Servers